CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Fix jwt error returning HTTP status `400` for invalid role by @taimoorzaeem in #3601
 
+### Changed
+
+- PostgREST now fails to start when `db-schemas` config is set to empty by @taimoorzaeem in #4089
```

Comment:
Hm, is this really a breaking change? We have this for [v13](https://github.com/PostgREST/postgrest/releases/tag/v13.0.0):

```
Fail to start on an invalid db-schemas or db-extra-search-path by @wolfgangwalther in https://github.com/PostgREST/postgrest/pull/3644
```

Isn't this a case of invalid `db-schemas`?

Reply:
> Isn't this a case of invalid `db-schemas`? If before we didn't fail on `<empty>`, then this should be a fix?

Hm, agreed. This is a fix.